galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/src/main/java/dev/galasa/docker/internal/properties/DockerRegistryPrefix.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.docker.internal.properties;
+
+import dev.galasa.docker.DockerManagerException;
+import dev.galasa.docker.internal.DockerRegistryImpl;
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+import dev.galasa.framework.spi.cps.CpsProperties;
+
+/**
+ * Docker Registry Prefix CPS Property
+ * 
+ * @galasa.cps.property
+ * 
+ * @galasa.name docker.registry.ID.prefix
+ * 
+ * @galasa.description Provides the prefix of a Docker Registry that is used by the Docker Manager.
```

Comment:
provides a prefix to be applied to all image names,   particularly useful if you have a dockerhub proxy